lib/abt_did.ex

```diff
@@ -226,15 +244,9 @@ defmodule AbtDid do
   # `:form`: Determines the form of DID returned. `:long` - The returned DID will be prefixed by "did:abt:". `:short` - The retuned DID has only DID string.
   # `:encode`: Detemines whether or not encode the DID. `true` - The returned DID will be encoded as Base58. `false` - The returned DID will be in binary format and `:form` will be set as `:short`.
   @spec pk_hash_to_did(DidType.t(), binary() | String.t(), Keyword.t()) :: String.t()
-  defp pk_hash_to_did(did_type, pk_hash, opts) do
-    pk_hash_bin =
-      if byte_size(pk_hash) == 40 and String.valid?(pk_hash) do
-        Base.decode16!(pk_hash, case: :mixed)
-      else
-        pk_hash
-      end
-
-    do_pk_hash_to_did(did_type, pk_hash_bin, opts)
+  defp pk_hash_to_did(did_type, hash, opts) do
```

Comment:
here looks nobody is using the check logic in this private function so I removed it.